tutorial/SparkRedshiftTutorial.scala

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright 2015 Databricks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.databricks.spark.redshift.tutorial
+
+import org.apache.spark.{ SparkConf, SparkContext }
+import org.apache.spark.sql.SaveMode
+import java.io.File
+import org.apache.spark.sql.types.{ StructType, StructField, 
+                                    StringType, IntegerType, 
+                                    LongType, DecimalType }
+
+/**
+ * Source code accompanying the spark-redshift tutorial. 
+ * The following parameters need to be passed
+ * 1. AWS Access Key
+ * 2. AWS Secret Access Key
+ * 3. Redshift Database Name
+ * 4. Redshift UserId
+ * 5. Redshift Password
+ * 6. Redshift URL (Ex. swredshift.czac2vcs84ci.us-east-1.redshift.amazonaws.com:5439
+ * 
+ */
+object SparkRedshiftTutorial {
+  /*
+   * For Windows Users only
+   * 1. Download contents from link 
+   *      https://github.com/srccodes/hadoop-common-2.2.0-bin/archive/master.zip
+   * 2. Unzip the file in step 1 into your %HADOOP_HOME%/bin. 
+   *        In the example below  %HADOOP_HOME%=C:/MySoftware/hdphome
+   * 3. After step 2 %HADOOP_HOME/bin should contain winutils.exe
+   * 
+   */   
+	if (System.getProperty("os.name").startsWith("Windows")) {
+		val hdphome: File = new File("C:/MySoftware/hdphome");
```

Comment:
Should this be hardcoded or should be it be loaded from the `HADOOP_HOME` environment variable?